packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -113,6 +113,45 @@ class FileCollection extends DocumentCollection {
     }
   }
 
+  /**
+   *  Add referenced_by documents to a file — see https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/#post-filesfile-idrelationshipsreferenced_by
```

Comment:
I think the docstrings should be a bit less concise, adding a small example like albums/playlists for example. I understand that most of the content is in the stack documentation but it is harder to navigate between docs than to stay in one documentation.